shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/domain/model/Question.kt

```diff
@@ -49,7 +51,9 @@ fun Question.displayType(): QuestionDisplayType {
             it.text ?: ""
         })
         QUESTION_DISPLAY_TYPE_NPS -> QuestionDisplayType.Nps
-        QUESTION_DISPLAY_TYPE_TEXTAREA -> QuestionDisplayType.Textarea
+        QUESTION_DISPLAY_TYPE_TEXTAREA -> QuestionDisplayType.Textarea(
+            sortedAnswers().first().inputMaskPlaceholder.orEmpty()
+        )
```

Comment:
Following my concern [here](https://github.com/suho/kmm-ic/pull/118#discussion_r1055069609), we can get the `first().inputMaskPlaceholder.orEmpty()` from `QuestionContentUiModel.answers` at SurveyQuestionsScreen. 🤔